### DISCUSSION TITLE
After running `atri start`, the terminal is stuck at 'Please wait while we get the server ready...' and the visual editor does not start

### DISCUSSION THREAD

**Discussion by darshitac11** [Atri-Labs/atrilabs-engine#331] [Help: Installation & Start] [answered]
Command: `atri start`

Error message:

`Please wait while we get the server ready...
`

Raised by @devsachin007, @Aashutosh-kr (Slack community) & @Aryan1982

**Comment by darshitac11** [comment]
@devsachin007 - Instead of `atri start`, please run `atri start --debug`.  This will produce a detailed log and will help us in identifying the issue.

**Reply by Piyush0102** [reply]
I am facing the same issue. Can you please check.

**Reply by darshitac11** [reply]
@Piyush0102 - Please share the Atri version that you are using.

**Comment by devsachin007** [comment]
Please wait while we get the server ready...Connection to ipc server failed!
Please wait while we get the server ready...Connection to ipc server failed!
Please wait while we get the server ready...Connection to ipc server failed!
Please wait while we get the server ready...Connection to ipc server failed!
Please wait while we get the server ready...Connection to ipc server failed!
Please wait while we get the server ready...Connection to ipc server failed!
Please wait while we get the server ready...


and so on

**Comment by darshitac11** [comment]
@devsachin007 - are you using 32 bit system by any chance?

**Comment by darshitac11** [comment]
After running `atri start debug`, we found that the build was failing because of `webapp builder`

System configuration of @Aashutosh-kr: Raspberry Pi 4 with Raspberry OS 32 bit 

However, the webapp builder is a 64 bit executable.

While it is possible to build `atri` for 32 bit, we do not recommend it. Our recommendation is to use 64 bit macOS, Linux or Windows.

<img width="982" alt="Screen Shot 2022-10-09 at 12 12 23 PM" src="https://user-images.githubusercontent.com/102641692/194767517-aa485a81-9fab-4ee7-b891-2a2ec819d1c7.png">

**Reply by Aashutosh-kr** [reply]
Can't I run 32 bit webapp builder, what limitations does it cause.

**Reply by darshitac11** [reply]
You can try, if you are interested but there might be other build failures down the line. 

The recent versions have only been tested in 64 bit. Hence it is not recommended.

**Reply by devsachin007** [reply]
no, my system is 64 and python is also 64 bit

**Reply by devsachin007** [reply]
by the way i am using pyCharm IDE not visual studio

**Reply by cruxcode** [reply]
IDE has not role in it!

**Comment by devsachin007** [comment]
didnt got any solution till now
i checked each and everything and all is good but engine not starting

**Reply by cruxcode** [reply]
Can you try running `pipenv` in your terminal and see if it works.

**Reply by cruxcode** [reply]
Which OS are you using?

**Reply by devsachin007** [reply]
Windows 10 64 bit

**Reply by devsachin007** [reply]
Yes in terminal pipenv command is working fine returning requirements already satisfied

**Reply by Aryan1982** [reply]
I am having the same issue, I am also using windows 10 64bit. Did you found any solution?

**Reply by darshitac11** [reply]
@Aryan1982 - Please share screenshot of your terminal after running `atri start --debug`.  

We have found one solution but want to be sure if it will work for you as well.

**Reply by devsachin007** [reply]
yes this is ss 
![Screenshot (275)](https://user-images.githubusercontent.com/79436926/196101178-a959d961-5af5-4b2e-be0c-7e2734a52cd0.png)

**Reply by cruxcode** [reply]
The fix will be available in `v0.0.73` which will be released today Oct 17, 2021, IST.

**Reply by cruxcode** [reply]
@devsachin007 we have released `v0.0.73`, you can either create a new project to start working or you can update atri in existing project by following steps here https://docs.atrilabs.com/existing-project/update-atri/

**Reply by devsachin007** [reply]
my atri engine started

**Comment by darshitac11** [comment, marked as answer]
Fixed in `v0.0.73`. Please update Atri version in existing project by following steps [here](https://docs.atrilabs.com/existing-project/update-atri/) or create a new project.

**Reply by Aryan1982** [reply]
Thank you! now it works fine.

**Reply by darshitac11** [reply]
Great, thanks!
